docs/src/tutorials.md

```diff
@@ -0,0 +1,16 @@
+# Tutorials
+
+## Installation
+
+Installation of the Element requires an integrated development environment and database.
+Instructions to setup each of the components can be found on the
+[User Instructions](datajoint.com/docs/elements/user-instructions) page. 
+
+There, you'll find instructions referencing other DataJoint Element 'Workflows' (e.g.,
+[workflow for Element DeepLabCut](https://github.com/datajoint/workflow-deeplabcut)),
```

Comment:
```suggestion
[workflow for Element Facemap](https://github.com/datajoint/workflow-facemap)),
```

Reply:
Yes, we don't currently have a `workflow-facemap`.